game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbem/EmailSenderEditor.java

```diff
@@ -177,6 +190,82 @@ public void viewModelChanged(final EmailSenderEditorViewModel viewModel) {
           toggleFieldVisibility();
           emailProviderSelectionBox.setSelectedItem(viewModel.getSelectedProvider());
 
+          if (viewModel.isEmailProviderDisabled()) {
+            helpMessage.setText(
+                "<html><p style='width: 400px;'>"
+                    + "Email sender<br/>"
+                    + "An email sender can email the turn summary and save game to multiple "
+                    + "recipients at the end of each players turn. This allows two or more "
+                    + "players to play a game where you don't have to be online at the same "
+                    + "time.<br/>"
+                    + "Each email sender may require custom configuration, to learn more "
+                    + "click help again after selecting a specific email sender."
+                    + "</p></html>");
+          } else if (viewModel.isEmailProviderGmail()) {
```

Comment:
Building on the last thread, this is different from a switch statement on a single value.

```
        } else if (viewModel.isEmailProviderGmail()) {
```
That is a brand new boolean condition on the view model, the view should be able to know what to do based on a single value. In this case we have business logic in the view.

We should be able to get rid of all if statements and the method `isEmailProviderGmail`. The view should not be aware of a concept of 'gmail' or the different view providers, only that it gets data and it knows how to render it.